charts/langgraph-dataplane/templates/operator/deployment.yaml

```diff
@@ -100,7 +100,7 @@ spec:
         {{- end }}
         env:
           - name: WATCH_NAMESPACE
-            value: {{ .Values.operator.watchNamespaces | quote }}
+            value: {{ default .Values.operator.watchNamespaces .Values.config.watchNamespaces | quote }}
```

Comment:
For backward compatibility, default to `.Values.operator.watchNamespaces` if `.Values.config.watchNamespaces` is not set.